about/release_policy.rst

```diff
@@ -0,0 +1,84 @@
+| **Version** | **Release date**  | **Support level**                                                        |
++-------------+-------------------+--------------------------------------------------------------------------+
+| Godot 4.0   | ~2021 (see below) | |unstable| *Current focus of development (unstable).*                    |
++-------------+-------------------+--------------------------------------------------------------------------+
+| Godot 3.2   | January 2020      | |supported| Backwards-compatible new features (backported from the       |
+|             |                   | ``master`` branch) as well as bug, security, and platform support fixes. |
++-------------+-------------------+--------------------------------------------------------------------------+
+| Godot 3.1   | March 2019        | |partial| Only security and platform support fixes.                      |
++-------------+-------------------+--------------------------------------------------------------------------+
+| Godot 3.0   | January 2018      | |partial| Only security and platform support fixes.                      |
++-------------+-------------------+--------------------------------------------------------------------------+
+| Godot 2.1   | July 2016         | |partial| Only security and platform support fixes.                      |
++-------------+-------------------+--------------------------------------------------------------------------+
+| Godot 2.0   | February 2016     | |eol| No longer supported.                                               |
++-------------+-------------------+--------------------------------------------------------------------------+
+| Godot 1.1   | May 2015          | |eol| No longer supported.                                               |
++-------------+-------------------+--------------------------------------------------------------------------+
+| Godot 1.0   | December 2014     | |eol| No longer supported.                                               |
++-------------+-------------------+--------------------------------------------------------------------------+
+
+.. |supported| image:: img/supported.png
+.. |partial| image:: img/partial.png
+.. |eol| image:: img/eol.png
+.. |unstable| image:: img/unstable.png
+
+**Legend:**
+|supported| Full support -
+|partial| Partial support -
+|eol| No support (end of life) -
+|unstable| Development version
+
+Pre-release Godot versions aren't intended to be used in production and are
+provided on a best-effort basis.
+
+When is the next release out?
+-----------------------------
+
+While Godot contributors aren't working under any deadlines, there's usually a
+major or minor Godot release made available every year. Following this trend,
+this means Godot 4.0 will most likely be released in **2021**.
+
+Patch releases are made available more frequently, typically every 2-6 months
```

Comment:
6 months is pretty extreme, I'd suggest `1-3 months` instead.

Reply:
As of me writing this reply, it has been 4 months since the release of Godot 3.2.3, which is more than 3 months.